Source/appfat.h

```diff
@@ -6,6 +6,12 @@
 #ifndef __APPFAT_H__
 #define __APPFAT_H__
 
+DEVILUTION_BEGIN_NAMESPACE

```

Comment:
Should we remove the namespace altogether as suggested by @glebm?

Reply:
Yes, I decided to merge the PR first so we can take it in steps. and it might require a bit more than just a search and replace :)

But removing it will make it simpler to do pulls from upstream (about 1/3 conflicts are from these lines (happens whenever there is a change to header or globals).